packages/calcite-components/src/components/card/card.scss

```diff
@@ -3,35 +3,27 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
  * @prop --calcite-card-background-color: Specifies the background color of the component.
+ * @prop --calcite-card-background-color-hover: Specifies the background color of elements in the component when hovered.
+ * @prop --calcite-card-background-color-press: Specifies the background color of elements in the component when pressed.
  * @prop --calcite-card-border-color: Specifies the border color of the component.
- * @prop --calcite-card-shadow: Specifies the shadow of the component.
  * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
- * @prop --calcite-card-accent-color-selected: Specifies the accent color of the component when `selected`.
- * @prop --calcite-card-selection-background-color: Specifies the background color of the component's selection element.
- * @prop --calcite-card-selection-background-color-hover: Specifies the background color of the component's selection element when hovered.
- * @prop --calcite-card-selection-background-color-active: Specifies the background color of the component's selection element when active.
- * @prop --calcite-card-selection-background-color-selected: Specifies the icon color of the component's selection element when `selected`.
- * @prop --calcite-card-selection-icon-color: Specifies the icon color of the component's selection element.
- * @prop --calcite-card-selection-icon-color-hover: Specifies the icon color of the component's selection element when hovered.
- * @prop --calcite-card-selection-icon-color-selected: Specifies the icon color of the component's selection element when `selected`.
+ * @prop --calcite-card-selection-color: Specifies the color of the component's selection element.
+ * @prop --calcite-card-selection-color-hover: Specifies the color of the component's selection element when hovered or focused.
+ * @prop --calcite-card-selection-background-color-active: [Deprecated] Use --calcite-card-background-color-press. Specifies the background color of the component's selection element when active.
+ * @prop --calcite-card-selection-background-color-hover: [Deprecated] --calcite-card-background-color-hover. Specifies the background color of the component's selection element when hovered.
+ * @prop --calcite-card-selection-background-color-selected: [Deprecated] Use --calcite-card-background-color. Specifies the icon color of the component's selection element when `selected`.
```

Comment:
Are these still intended to be deprecated? 

I would not expect  to use “—calcite-card-background-color” to control the background color of the internal “checkbox  element” like this deprecation message suggests.

The newly introduced names are confusing to me, there is no general hover or press for a Card component itself, those states should be applied separately via css prop imo. If a user wants to set calcite-card:hover { whatever } they can do so, but this internally rendered element is different.

Reply:
It's not actually a checkbox component - it's just a styled div with an icon as it is in most other selection-enabled components.

The design I posted above with the multiple selection cards is a real-world use case I'd like to achieve (and is currently possible) - so removing these would prevent that design. We shouldn't be arbiters of theming choices - if we are truly making our components fully them able via tokens, everything should be theme able, not just what some deem appropriate.

I think this proposed nomenclature is more confusing - there is no hover state for the card - and we want folks to use stateful overrides (:hover, etc.) for actual parent-level adjustments based on those interactions right? 

IMO setting `--calcite-card-background-color-hover` to control the bg of a portion of the card doesn't make sense.

Reply:
> if we are truly making our components fully them able via tokens, everything should be theme able, not just what some deem appropriate.

This is not what we have discussed as a team. What we are providing is white labeling. We still have design patterns we have set and want to adhere to. That said, I can understand your argument against `--calcite-card-background-color-hover`, I've updated the tokens. Is this an acceptable compromise?

Reply:
Sounds good